NetworkPkg/Ip6Dxe/Ip6Impl.c

```diff
@@ -67,6 +67,7 @@ EfiIp6GetModeData (
   IpIf       = IpInstance->Interface;
 
   if (IpSb->LinkLocalDadFail) {
+    gBS->RestoreTPL (OldTpl);

```

Comment:
I think this can this follow the same pattern as the other 2 fixes and simply move RaiseTPL() call below this if statement.  The lines above are only doing pointer math and reads with no side effects.

Reply:
Updated.